gnd/src/main/res/values/strings.xml

```diff
@@ -15,11 +15,14 @@
   -->
 
 <resources>
+  <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">AIzaSyAHk3dTs0hYeFb09pqq_rIw0I67CbT8ch0</string>
```

Comment:
Looks like you accidentally shared your Maps API key. If you instead add it to a separate file as described [here](https://github.com/google/ground-android#add-google-maps-api-keys), it will be excluded from Git by our .gitignore file.